addons/sale_timesheet/models/product.py

```diff
@@ -157,9 +157,14 @@ def _is_delivered_timesheet(self):
         self.ensure_one()
         return self.type == 'service' and self.service_policy == 'delivered_timesheet'
 
+    def _inverse_service_policy(self):
+        for product in self:
+            if product.service_policy:
+                product.invoice_policy, product.service_type = SERVICE_TO_GENERAL.get(product.service_policy, (False, False))
```

Comment:
Since the inverse method of project.template is defined in sale_project, we should defined it in the same same module. 🙂 

Can you move that method in sale_project module? 🙂 

Thanks in advance! 🙂

Reply:
it's in sale_timesheet no ?
https://github.com/odoo/odoo/blob/40449f2170b4e753569661fed80d86c7ddbd73ff/addons/sale_timesheet/models/product.py#L57-L61

Reply:
indeed but in 16.0, we will have to move it in sale_project since the other inverse method moved. 🙂

Reply:
yes, i will resolve it during the fw